src/simplewallet/simplewallet.cpp

```diff
@@ -4527,6 +4529,39 @@ bool simple_wallet::get_tx_note(const std::vector<std::string> &args)
   return true;
 }
 //----------------------------------------------------------------------------------------------------
+bool simple_wallet::set_description(const std::vector<std::string> &args)
+{
+  // 0 arguments allowed, for setting the description to empty string
+
+  std::string description = "";
+  for (size_t n = 0; n < args.size(); ++n)
+  {
+    if (n > 0)
+      description += " ";
+    description += args[n];
+  }
```

Comment:
For the record, I think this whole loop could be: std::string description = boost::join(args, " ");
Not asking for it to change though, this is clear/simple enough.

Reply:
I see. Maybe somebody can optimize this later, here and in the TX note code where I stole those lines :)